pkg/helm/helm.go

```diff
@@ -0,0 +1,20 @@
+package helm
```

Comment:
I'm not completely sure this is the right package for this method. I would expect this to be in cmd/config or in other kind of "port" package. This `helm` package should be agnostic from how the user and password are read. In fact there could potentially be multiple ways to read it, depending on what's the interface/port for this codepath.